### PR TITLE
Change 3D Secure to 3D Secure 2

### PR DESCRIPTION
**Pull request by jcruzbam**
BIC-4450 Change 3D Secure to 3D Secure 2.0 in developer guide

### CODE REVIEW

**Review by chaganiA — CHANGES_REQUESTED** [review]
replied in #nam-rtp-pr channel

File: source/docs/guides/3D_secure_2_0.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: EMV 3D Secure
+title: EMV 3D Secure 2.0
```
Comment (chaganiA) [inline source/docs/guides/3D_secure_2_0.md]: Text need to be updated here as well.

File: source/docs/guides/index.md [hunk 2]
```diff
@@ -42,11 +42,11 @@ cards:
         icon: list
         link: /docs/guides/3D_secure
     -
-        title: EMV 3D Secure
+        title: 3D Secure 2.0
         description: >
             Learn about implementing EMV 3D Secure based payments.
```
Comment (chaganiA) [inline source/docs/guides/index.md]: here as well?

**Review by chaganiA — APPROVED** [review]

**Review by jgalvanbam — APPROVED** [review]
Looks good

**Pull request merged**